201-vm-efficientip-vhd/azuredeploy.json

```diff
@@ -0,0 +1,186 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the VM"
+      }
+    },
+    "osType": {
+      "type": "string",
+      "allowedValues": [
+        "Linux"
+      ],
+      "metadata": {
+        "description": "Type of OS on the existing vhd"
+      }
+    },
+    "osDiskVhdUri": {
+      "type": "string",
+      "metadata": {
+        "description": "URI of the existing VHD in ARM standard or premium storage"
+      }
+    },
+    "vmSize": {
+      "type": "string",
+	  "allowedValues": [
+        "Standard_D2_v2"
+	  ],
+      "metadata": {
+        "description": "Size of the VM"
+      }
+    },
+	"vmDiskSizeGB": {
+        "type": "string",
+         "allowedValues": [
+            "150","300"
+	    ],
+        "metadata": {
+          "description": "Diskspace size of the VM"  
+        }
+    },
+    "existingVirtualNetworkName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the existing VNET"
+      }
+    },
+    "existingVirtualNetworkResourceGroup": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the existing VNET resource group"
+      }
+    },
+    "subnetName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the subnet in the virtual network you want to use"
+      }
+    },
+	"existingSecurityGroupName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the network Security Group that needs to be assocated to virtual NIC. "
+      }
+    },
+    "dnsNameForPublicIP": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+      }
+    }
+  },
+  "variables": {
+    "diagStorageAccountName": "[concat(uniquestring(resourceGroup().id), 'specvm')]",
+    "api-version": "2015-06-15",
```

Comment:
remove this and use a literal apiVersion

Reply:
I am not sure I understand.  This is a API version.  What is wrong with this version?